{
  "name": "mbpp_268_find_star_num",
  "language": "php",
  "prompt": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n",
  "doctests": "keep",
  "original": "/home/arjun/repos/MultiPL-E/datasets/../datasets/mbpp-typed/mbpp_268_find_star_num.py",
  "prompt_terminology": "verbatim",
  "tests": "function candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
  "stop_tokens": [
    "\nfunction",
    "\n?>",
    "\n//",
    "\n#"
  ],
  "results": [
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp313yyb1x.php:19\nStack trace:\n#0 /tmp/tmp313yyb1x.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp313yyb1x.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpsqy1fzwm.php:16\nStack trace:\n#0 /tmp/tmpsqy1fzwm.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpsqy1fzwm.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num += $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpp4h_ayin.php:16\nStack trace:\n#0 /tmp/tmpp4h_ayin.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpp4h_ayin.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    while ($n > 1) {\n        $star_num = $star_num * 2 + 1;\n        $n--;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp5b4aclnt.php:17\nStack trace:\n#0 /tmp/tmp5b4aclnt.php(22): test()\n#1 {main}\n  thrown in /tmp/tmp5b4aclnt.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp313yyb1x.php:19\nStack trace:\n#0 /tmp/tmp313yyb1x.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp313yyb1x.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $count = 0;\n    while ($count < $n) {\n        $i++;\n        if (is_star_num($i)) {\n            $count++;\n        }\n    }\n    return $i;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmpgeqbye7t.php:8\nStack trace:\n#0 /tmp/tmpgeqbye7t.php(16): find_star_num(3)\n#1 /tmp/tmpgeqbye7t.php(20): candidate(3)\n#2 /tmp/tmpgeqbye7t.php(25): test()\n#3 {main}\n  thrown in /tmp/tmpgeqbye7t.php on line 8\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp313yyb1x.php:19\nStack trace:\n#0 /tmp/tmp313yyb1x.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp313yyb1x.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 0;\n    $j = 0;\n    while ($i < $n) {\n        $j++;\n        if (is_star_num($j)) {\n            $i++;\n        }\n    }\n    return $j;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmp23_bcb8h.php:8\nStack trace:\n#0 /tmp/tmp23_bcb8h.php(16): find_star_num(3)\n#1 /tmp/tmp23_bcb8h.php(20): candidate(3)\n#2 /tmp/tmp23_bcb8h.php(25): test()\n#3 {main}\n  thrown in /tmp/tmp23_bcb8h.php on line 8\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n  $star_num = 0;\n  for ($i = 1; $i <= $n; $i++) {\n    $star_num = $star_num + $i;\n  }\n  return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp9pt26xi5.php:17\nStack trace:\n#0 /tmp/tmp9pt26xi5.php(22): test()\n#1 {main}\n  thrown in /tmp/tmp9pt26xi5.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpsqy1fzwm.php:16\nStack trace:\n#0 /tmp/tmpsqy1fzwm.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpsqy1fzwm.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpsqy1fzwm.php:16\nStack trace:\n#0 /tmp/tmpsqy1fzwm.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpsqy1fzwm.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $num = $num * 2 + 1;\n    }\n    return $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp34yvctpw.php:16\nStack trace:\n#0 /tmp/tmp34yvctpw.php(21): test()\n#1 {main}\n  thrown in /tmp/tmp34yvctpw.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $count = 0;\n    while ($count < $n) {\n        $i++;\n        if (is_star_num($i)) {\n            $count++;\n        }\n    }\n    return $i;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmpgeqbye7t.php:8\nStack trace:\n#0 /tmp/tmpgeqbye7t.php(16): find_star_num(3)\n#1 /tmp/tmpgeqbye7t.php(20): candidate(3)\n#2 /tmp/tmpgeqbye7t.php(25): test()\n#3 {main}\n  thrown in /tmp/tmpgeqbye7t.php on line 8\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpsqy1fzwm.php:16\nStack trace:\n#0 /tmp/tmpsqy1fzwm.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpsqy1fzwm.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $count = 0;\n    while ($count < $n) {\n        $i++;\n        if (is_star_num($i)) {\n            $count++;\n        }\n    }\n    return $i;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmpgeqbye7t.php:8\nStack trace:\n#0 /tmp/tmpgeqbye7t.php(16): find_star_num(3)\n#1 /tmp/tmpgeqbye7t.php(20): candidate(3)\n#2 /tmp/tmpgeqbye7t.php(25): test()\n#3 {main}\n  thrown in /tmp/tmpgeqbye7t.php on line 8\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num += $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpp4h_ayin.php:16\nStack trace:\n#0 /tmp/tmpp4h_ayin.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpp4h_ayin.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $count = 0;\n    while ($count < $n) {\n        if (is_star_num($i)) {\n            $count++;\n        }\n        $i++;\n    }\n    return $i - 1;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmp_o2f5e5n.php:7\nStack trace:\n#0 /tmp/tmp_o2f5e5n.php(16): find_star_num(3)\n#1 /tmp/tmp_o2f5e5n.php(20): candidate(3)\n#2 /tmp/tmp_o2f5e5n.php(25): test()\n#3 {main}\n  thrown in /tmp/tmp_o2f5e5n.php on line 7\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "12345\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp4f_n3hru.php:17\nStack trace:\n#0 /tmp/tmp4f_n3hru.php(22): test()\n#1 {main}\n  thrown in /tmp/tmp4f_n3hru.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $count = 0;\n    $num = 0;\n    while($count < $n) {\n        $num++;\n        if(is_star_num($num)) {\n            $count++;\n        }\n    }\n    return $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmpfeef_f0j.php:8\nStack trace:\n#0 /tmp/tmpfeef_f0j.php(16): find_star_num(3)\n#1 /tmp/tmpfeef_f0j.php(20): candidate(3)\n#2 /tmp/tmpfeef_f0j.php(25): test()\n#3 {main}\n  thrown in /tmp/tmpfeef_f0j.php on line 8\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\n$n = 5;\necho find_star_num($n);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "12345\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp_s7odod7.php:18\nStack trace:\n#0 /tmp/tmp_s7odod7.php(23): test()\n#1 {main}\n  thrown in /tmp/tmp_s7odod7.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(3);\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "123\nFatal error: Uncaught Exception: Test failed! in /tmp/tmplw_dc8ku.php:18\nStack trace:\n#0 /tmp/tmplw_dc8ku.php(23): test()\n#1 {main}\n  thrown in /tmp/tmplw_dc8ku.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    $i = 1;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpy7q6wsrh.php:18\nStack trace:\n#0 /tmp/tmpy7q6wsrh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpy7q6wsrh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    $i = 1;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpy7q6wsrh.php:18\nStack trace:\n#0 /tmp/tmpy7q6wsrh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpy7q6wsrh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $num = $num * 10 + $i;\n    }\n    return $num;\n}\n\necho find_star_num(3);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "123\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp8sny6ofa.php:17\nStack trace:\n#0 /tmp/tmp8sny6ofa.php(22): test()\n#1 {main}\n  thrown in /tmp/tmp8sny6ofa.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num += $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\necho \"<br>\";\necho find_star_num(10);\necho \"<br>\";\necho find_star_num(15);\necho \"<br>\";\necho find_star_num(20);\necho \"<br>\";\necho find_star_num(25);\necho \"<br>\";\necho find_star_num(30);\necho \"<br>\";\necho find_star_num(35);\necho \"<br>\";\necho find_star_num(40);\necho \"<br>\";\necho find_star_num(45);\necho \"<br>\";\necho find_star_num(50);\necho \"<br>\";\necho find_star_num(55);\necho \"<br>\";\necho find_star_num(60);\necho \"<br>\";\necho find_star_num(65);\necho \"<br>\";\necho find_star_num(70);\necho \"<br>\";\necho find_star_num(75);\necho \"<br>\";\necho find_star_num(80);\necho \"<br>\";\necho find_star_num(85);\necho \"<br>\";\necho find_star_num(90);\necho \"<br>\";\necho find_star_num(95);\necho \"<br>\";\necho find_star_num(100);\necho \"<br>\";\necho find_star_num(105);\necho \"<br>\";\necho find_star_num(110);\necho \"<br>\";\necho find_star_num(115);\necho \"<br>\";\necho find_star_num(120);\necho \"<br>\";\necho find_star_num(125);\necho \"<br>\";\necho find_star_num(130);\necho \"<br>\";\necho find_star_num(135);\necho \"<br>\";\necho find_star_num(140);\necho \"<br>\";\necho find_star_num(145);\necho \"<\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nParse error: syntax error, unexpected identifier \"Test\", expecting \",\" or \";\" in /tmp/tmp1rldv_q9.php on line 74\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    $i = 1;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmphyq6hzed.php:19\nStack trace:\n#0 /tmp/tmphyq6hzed.php(24): test()\n#1 {main}\n  thrown in /tmp/tmphyq6hzed.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n  $num = 1;\n  while ($n > 0) {\n    $num += 2;\n    $count = 0;\n    for ($i = 1; $i <= $num; $i++) {\n      if ($num % $i == 0) {\n        $count++;\n      }\n    }\n    if ($count == 2) {\n      $n--;\n    }\n  }\n  return $num;\n}\n\necho find_star_num(1) . \"\\n\";\necho find_star_num(2) . \"\\n\";\necho find_star_num(3) . \"\\n\";\necho find_star_num(4) . \"\\n\";\necho find_star_num(5) . \"\\n\";\necho find_star_num(6) . \"\\n\";\necho find_star_num(7) . \"\\n\";\necho find_star_num(8) . \"\\n\";\necho find_star_num(9) . \"\\n\";\necho find_star_num(10) . \"\\n\";\necho find_star_num(11) . \"\\n\";\necho find_star_num(12) . \"\\n\";\necho find_star_num(13) . \"\\n\";\necho find_star_num(14) . \"\\n\";\necho find_star_num(15) . \"\\n\";\necho find_star_num(16) . \"\\n\";\necho find_star_num(17) . \"\\n\";\necho find_star_num(18) . \"\\n\";\necho find_star_num(19) . \"\\n\";\necho find_star_num(20) . \"\\n\";\necho find_star_num(21) . \"\\n\";\necho find_star_num(22) . \"\\n\";\necho find_star_num(23) . \"\\n\";\necho find_star_num(24) . \"\\n\";\necho find_star_num(25) . \"\\n\";\necho find_star_num(26) . \"\\n\";\necho find_star_num(27) . \"\\n\";\necho find_star_num(28) . \"\\n\";\necho find_star_num(29) . \"\\n\";\necho find_star\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nParse error: syntax error, unexpected token \"function\", expecting \",\" or \";\" in /tmp/tmp7pimngvx.php on line 50\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp313yyb1x.php:19\nStack trace:\n#0 /tmp/tmp313yyb1x.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp313yyb1x.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $j = 1;\n    $k = 1;\n    $star_num = 0;\n    while ($k <= $n) {\n        $star_num = $i * $j;\n        $i++;\n        $j++;\n        $k++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "25\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp9ihx_s_z.php:23\nStack trace:\n#0 /tmp/tmp9ihx_s_z.php(28): test()\n#1 {main}\n  thrown in /tmp/tmp9ihx_s_z.php on line 23\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n\t$star_num = 1;\n\twhile ($n > 1) {\n\t\t$star_num = $star_num * 2 + 1;\n\t\t$n--;\n\t}\n\treturn $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpidlw_rg1.php:17\nStack trace:\n#0 /tmp/tmpidlw_rg1.php(22): test()\n#1 {main}\n  thrown in /tmp/tmpidlw_rg1.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = ($star_num * 10) + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp7eb7gb7n.php:16\nStack trace:\n#0 /tmp/tmp7eb7gb7n.php(21): test()\n#1 {main}\n  thrown in /tmp/tmp7eb7gb7n.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "12345\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp4f_n3hru.php:17\nStack trace:\n#0 /tmp/tmp4f_n3hru.php(22): test()\n#1 {main}\n  thrown in /tmp/tmp4f_n3hru.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n\t$num = 1;\n\tfor ($i = 1; $i <= $n; $i++) {\n\t\t$num = $num * 2 + 1;\n\t}\n\treturn $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpaoz563ea.php:16\nStack trace:\n#0 /tmp/tmpaoz563ea.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpaoz563ea.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    if ($n == 1) {\n        return 1;\n    }\n    return find_star_num($n - 1) + $n;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd1f0fyun.php:16\nStack trace:\n#0 /tmp/tmpd1f0fyun.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpd1f0fyun.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    while ($n > 1) {\n        $star_num = $star_num * 2 + 1;\n        $n--;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp5b4aclnt.php:17\nStack trace:\n#0 /tmp/tmp5b4aclnt.php(22): test()\n#1 {main}\n  thrown in /tmp/tmp5b4aclnt.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 1;\n    $count = 1;\n    while ($count < $n) {\n        $num++;\n        $num_str = (string)$num;\n        $num_str_len = strlen($num_str);\n        $num_str_len_half = (int)($num_str_len / 2);\n        $num_str_len_half_ceil = ceil($num_str_len / 2);\n        $num_str_len_half_floor = floor($num_str_len / 2);\n        $num_str_half_left = substr($num_str, 0, $num_str_len_half);\n        $num_str_half_right = substr($num_str, $num_str_len_half_ceil);\n        if ($num_str_half_left == $num_str_half_right) {\n            $count++;\n        }\n    }\n    return $num;\n}\n\necho find_star_num(1) . PHP_EOL;\necho find_star_num(2) . PHP_EOL;\necho find_star_num(3) . PHP_EOL;\necho find_star_num(4) . PHP_EOL;\necho find_star_num(5) . PHP_EOL;\necho find_star_num(6) . PHP_EOL;\necho find_star_num(7) . PHP_EOL;\necho find_star_num(8) . PHP_EOL;\necho find_star_num(9) . PHP_EOL;\necho find_star_num(10) . PHP_EOL;\necho find_star_num(11) . PHP_EOL;\necho find_star_num(12) . PHP_EOL;\necho find_star_num(13) . PHP_EOL;\necho find_star_num(14) . PHP_EOL;\necho find_star_num(15) . PHP_EOL;\necho find_star_num(16) . PHP_EOL;\necho find_star_num(17) . PHP_EOL;\necho find_star_num(18) . PHP\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nParse error: syntax error, unexpected token \"function\", expecting \",\" or \";\" in /tmp/tmpcb58dlkw.php on line 40\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num += $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpp4h_ayin.php:16\nStack trace:\n#0 /tmp/tmpp4h_ayin.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpp4h_ayin.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num = $star_num + $i;\n        $i++;\n    }\n    return $star_num;\n}\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp2fbrfrmk.php:18\nStack trace:\n#0 /tmp/tmp2fbrfrmk.php(23): test()\n#1 {main}\n  thrown in /tmp/tmp2fbrfrmk.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    while ($n > 0) {\n        $i++;\n        $count = 0;\n        for ($j = 1; $j <= $i; $j++) {\n            if ($i % $j == 0) {\n                $count++;\n            }\n        }\n        if ($count == 2) {\n            $n--;\n        }\n    }\n    return $i;\n}\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    $i = 1;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpy7q6wsrh.php:18\nStack trace:\n#0 /tmp/tmpy7q6wsrh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpy7q6wsrh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpib4hctv3.php:18\nStack trace:\n#0 /tmp/tmpib4hctv3.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpib4hctv3.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    $i = 1;\n    while ($i < $n) {\n        $star_num = $star_num + 6 * ($i - 1);\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpt5vcqofz.php:18\nStack trace:\n#0 /tmp/tmpt5vcqofz.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpt5vcqofz.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpfwa20ins.php:16\nStack trace:\n#0 /tmp/tmpfwa20ins.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpfwa20ins.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    $i = 1;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpy7q6wsrh.php:18\nStack trace:\n#0 /tmp/tmpy7q6wsrh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpy7q6wsrh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 1;\n    $count = 0;\n    while ($count < $n) {\n        $num++;\n        if (is_star_num($num)) {\n            $count++;\n        }\n    }\n    return $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmp857_8um7.php:8\nStack trace:\n#0 /tmp/tmp857_8um7.php(16): find_star_num(3)\n#1 /tmp/tmp857_8um7.php(20): candidate(3)\n#2 /tmp/tmp857_8um7.php(25): test()\n#3 {main}\n  thrown in /tmp/tmp857_8um7.php on line 8\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num += $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpfrlvvf4p.php:17\nStack trace:\n#0 /tmp/tmpfrlvvf4p.php(22): test()\n#1 {main}\n  thrown in /tmp/tmpfrlvvf4p.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    $i = 1;\n    while ($i < $n) {\n        $star_num = $star_num + 6 * $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpqfg201hb.php:18\nStack trace:\n#0 /tmp/tmpqfg201hb.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpqfg201hb.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 1;\n    $count = 0;\n    while ($count < $n) {\n        $num++;\n        if (is_star_num($num)) {\n            $count++;\n        }\n    }\n    return $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmp857_8um7.php:8\nStack trace:\n#0 /tmp/tmp857_8um7.php(16): find_star_num(3)\n#1 /tmp/tmp857_8um7.php(20): candidate(3)\n#2 /tmp/tmp857_8um7.php(25): test()\n#3 {main}\n  thrown in /tmp/tmp857_8um7.php on line 8\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    $i = 1;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\n$n = 5;\necho find_star_num($n);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpct2ulu01.php:20\nStack trace:\n#0 /tmp/tmpct2ulu01.php(25): test()\n#1 {main}\n  thrown in /tmp/tmpct2ulu01.php on line 20\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num + 4 * $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmphu4tkl9y.php:16\nStack trace:\n#0 /tmp/tmphu4tkl9y.php(21): test()\n#1 {main}\n  thrown in /tmp/tmphu4tkl9y.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num += $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpp4h_ayin.php:16\nStack trace:\n#0 /tmp/tmpp4h_ayin.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpp4h_ayin.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\n$n = 5;\necho find_star_num($n);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "12345\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpu9mpws1f.php:18\nStack trace:\n#0 /tmp/tmpu9mpws1f.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpu9mpws1f.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $num = 0;\n    while ($i <= $n) {\n        $num += $i;\n        $i++;\n    }\n    return $num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp6blpael8.php:19\nStack trace:\n#0 /tmp/tmp6blpael8.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp6blpael8.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $num = ($num * 10) + $i;\n    }\n    return $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmphocdl27a.php:16\nStack trace:\n#0 /tmp/tmphocdl27a.php(21): test()\n#1 {main}\n  thrown in /tmp/tmphocdl27a.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpsqy1fzwm.php:16\nStack trace:\n#0 /tmp/tmpsqy1fzwm.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpsqy1fzwm.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    $count = 1;\n    while ($count < $n) {\n        $star_num += 2;\n        $count++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpcpjwx8c7.php:18\nStack trace:\n#0 /tmp/tmpcpjwx8c7.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpcpjwx8c7.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num = $star_num + $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp6tv6875l.php:19\nStack trace:\n#0 /tmp/tmp6tv6875l.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp6tv6875l.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 1;\n    $count = 0;\n    while ($count < $n) {\n        $num++;\n        if (is_star_num($num)) {\n            $count++;\n        }\n    }\n    return $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmp857_8um7.php:8\nStack trace:\n#0 /tmp/tmp857_8um7.php(16): find_star_num(3)\n#1 /tmp/tmp857_8um7.php(20): candidate(3)\n#2 /tmp/tmp857_8um7.php(25): test()\n#3 {main}\n  thrown in /tmp/tmp857_8um7.php on line 8\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    $i = 1;\n    while ($i < $n) {\n        $star_num += 6 * $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp4g7bhz1b.php:18\nStack trace:\n#0 /tmp/tmp4g7bhz1b.php(23): test()\n#1 {main}\n  thrown in /tmp/tmp4g7bhz1b.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpsqy1fzwm.php:16\nStack trace:\n#0 /tmp/tmpsqy1fzwm.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpsqy1fzwm.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    $i = 1;\n    while ($i < $n) {\n        $star_num += 6 * $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "61\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpbmeqiwg0.php:19\nStack trace:\n#0 /tmp/tmpbmeqiwg0.php(24): test()\n#1 {main}\n  thrown in /tmp/tmpbmeqiwg0.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $result = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $result += $i;\n    }\n    return $result;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpjele_uhn.php:16\nStack trace:\n#0 /tmp/tmpjele_uhn.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpjele_uhn.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 1;\n    while ($i <= $n) {\n        $star_num = $star_num + $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpjhh5wfvs.php:18\nStack trace:\n#0 /tmp/tmpjhh5wfvs.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpjhh5wfvs.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 0; $i < $n; $i++) {\n        $star_num = ($star_num * 10) + 9;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmppdsat6k1.php:16\nStack trace:\n#0 /tmp/tmppdsat6k1.php(21): test()\n#1 {main}\n  thrown in /tmp/tmppdsat6k1.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp313yyb1x.php:19\nStack trace:\n#0 /tmp/tmp313yyb1x.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp313yyb1x.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(3);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "123\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpowi88mhp.php:17\nStack trace:\n#0 /tmp/tmpowi88mhp.php(22): test()\n#1 {main}\n  thrown in /tmp/tmpowi88mhp.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp313yyb1x.php:19\nStack trace:\n#0 /tmp/tmp313yyb1x.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp313yyb1x.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    $num = 1;\n    while ($n > 0) {\n        $num = $num + 2;\n        $star_num = $star_num + $num;\n        $n--;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpp_7m2y4w.php:19\nStack trace:\n#0 /tmp/tmpp_7m2y4w.php(24): test()\n#1 {main}\n  thrown in /tmp/tmpp_7m2y4w.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n\t$star_num = 1;\n\tfor ($i = 1; $i < $n; $i++) {\n\t\t$star_num = $star_num + (6 * $i);\n\t}\n\treturn $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpa0ra87f8.php:16\nStack trace:\n#0 /tmp/tmpa0ra87f8.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpa0ra87f8.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    if ($n == 1) {\n        $star_num = 1;\n    } else {\n        $star_num = find_star_num($n - 1) + 6 * ($n - 1);\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "61\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpoiafrgwx.php:19\nStack trace:\n#0 /tmp/tmpoiafrgwx.php(24): test()\n#1 {main}\n  thrown in /tmp/tmpoiafrgwx.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "12345\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpq3e_ymz8.php:17\nStack trace:\n#0 /tmp/tmpq3e_ymz8.php(22): test()\n#1 {main}\n  thrown in /tmp/tmpq3e_ymz8.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n  $star_num = 0;\n  $i = 1;\n  while ($i <= $n) {\n    $star_num += $i;\n    $i++;\n  }\n  return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp7q6apmfw.php:18\nStack trace:\n#0 /tmp/tmp7q6apmfw.php(23): test()\n#1 {main}\n  thrown in /tmp/tmp7q6apmfw.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpsqy1fzwm.php:16\nStack trace:\n#0 /tmp/tmpsqy1fzwm.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpsqy1fzwm.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "12345\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp4f_n3hru.php:17\nStack trace:\n#0 /tmp/tmp4f_n3hru.php(22): test()\n#1 {main}\n  thrown in /tmp/tmp4f_n3hru.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num + (6 * ($i - 1));\n    }\n    return $star_num;\n}\n\necho find_star_num(3);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "7\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpqzxri73u.php:17\nStack trace:\n#0 /tmp/tmpqzxri73u.php(22): test()\n#1 {main}\n  thrown in /tmp/tmpqzxri73u.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 0;\n    for($i = 1; $i <= $n; $i++) {\n        $num = (2 * $num) + 1;\n    }\n    return $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnee46xa6.php:16\nStack trace:\n#0 /tmp/tmpnee46xa6.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnee46xa6.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = ($star_num * 10) + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp7eb7gb7n.php:16\nStack trace:\n#0 /tmp/tmp7eb7gb7n.php(21): test()\n#1 {main}\n  thrown in /tmp/tmp7eb7gb7n.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $i * ($i + 1);\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "30\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp91y0v0gy.php:17\nStack trace:\n#0 /tmp/tmp91y0v0gy.php(22): test()\n#1 {main}\n  thrown in /tmp/tmp91y0v0gy.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $count = 0;\n    while ($count < $n) {\n        if (is_star_num($i)) {\n            $count++;\n        }\n        $i++;\n    }\n    return $i - 1;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmp_o2f5e5n.php:7\nStack trace:\n#0 /tmp/tmp_o2f5e5n.php(16): find_star_num(3)\n#1 /tmp/tmp_o2f5e5n.php(20): candidate(3)\n#2 /tmp/tmp_o2f5e5n.php(25): test()\n#3 {main}\n  thrown in /tmp/tmp_o2f5e5n.php on line 7\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $j = 1;\n    $k = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num = $j * $k;\n        $j++;\n        $k++;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(3);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "9\nFatal error: Uncaught Exception: Test failed! in /tmp/tmprtjsycht.php:23\nStack trace:\n#0 /tmp/tmprtjsycht.php(28): test()\n#1 {main}\n  thrown in /tmp/tmprtjsycht.php on line 23\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n  $star_num = 0;\n  for ($i = 1; $i <= $n; $i++) {\n    $star_num = ($star_num * 10) + $i;\n  }\n  return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "12345\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpum5y0il4.php:17\nStack trace:\n#0 /tmp/tmpum5y0il4.php(22): test()\n#1 {main}\n  thrown in /tmp/tmpum5y0il4.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $count = 0;\n    while ($count < $n) {\n        if (is_star_num($i)) {\n            $count++;\n        }\n        $i++;\n    }\n    return $i - 1;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmp_o2f5e5n.php:7\nStack trace:\n#0 /tmp/tmp_o2f5e5n.php(16): find_star_num(3)\n#1 /tmp/tmp_o2f5e5n.php(20): candidate(3)\n#2 /tmp/tmp_o2f5e5n.php(25): test()\n#3 {main}\n  thrown in /tmp/tmp_o2f5e5n.php on line 7\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num += $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpp4h_ayin.php:16\nStack trace:\n#0 /tmp/tmpp4h_ayin.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpp4h_ayin.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "12345\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp4f_n3hru.php:17\nStack trace:\n#0 /tmp/tmp4f_n3hru.php(22): test()\n#1 {main}\n  thrown in /tmp/tmp4f_n3hru.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n  $star_num = 1;\n  for ($i = 1; $i <= $n; $i++) {\n    $star_num = $star_num * 2 + 1;\n  }\n  return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpxpm98cr1.php:16\nStack trace:\n#0 /tmp/tmpxpm98cr1.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpxpm98cr1.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = ($star_num * 10) + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp7eb7gb7n.php:16\nStack trace:\n#0 /tmp/tmp7eb7gb7n.php(21): test()\n#1 {main}\n  thrown in /tmp/tmp7eb7gb7n.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    $i = 1;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmphyq6hzed.php:19\nStack trace:\n#0 /tmp/tmphyq6hzed.php(24): test()\n#1 {main}\n  thrown in /tmp/tmphyq6hzed.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n  $star_num = 0;\n  for ($i = 1; $i <= $n; $i++) {\n    $star_num = ($star_num * 10) + $i;\n  }\n  return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpyg2oax23.php:16\nStack trace:\n#0 /tmp/tmpyg2oax23.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpyg2oax23.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num = $star_num + $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp6tv6875l.php:19\nStack trace:\n#0 /tmp/tmp6tv6875l.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp6tv6875l.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $count = 0;\n    while ($count < $n) {\n        $i++;\n        if (is_star_num($i)) {\n            $count++;\n        }\n    }\n    return $i;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmpgeqbye7t.php:8\nStack trace:\n#0 /tmp/tmpgeqbye7t.php(16): find_star_num(3)\n#1 /tmp/tmpgeqbye7t.php(20): candidate(3)\n#2 /tmp/tmpgeqbye7t.php(25): test()\n#3 {main}\n  thrown in /tmp/tmpgeqbye7t.php on line 8\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $num = $num * 10 + $i;\n    }\n    return $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpv7r6c47w.php:16\nStack trace:\n#0 /tmp/tmpv7r6c47w.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpv7r6c47w.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 1;\n    $count = 1;\n    while ($count < $n) {\n        $num++;\n        $is_star = true;\n        for ($i = 2; $i <= sqrt($num); $i++) {\n            if ($num % $i == 0) {\n                $is_star = false;\n                break;\n            }\n        }\n        if ($is_star) {\n            $count++;\n        }\n    }\n    return $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp2tuo9sz3.php:27\nStack trace:\n#0 /tmp/tmp2tuo9sz3.php(32): test()\n#1 {main}\n  thrown in /tmp/tmp2tuo9sz3.php on line 27\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num = $star_num + $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp1mq05vni.php:18\nStack trace:\n#0 /tmp/tmp1mq05vni.php(23): test()\n#1 {main}\n  thrown in /tmp/tmp1mq05vni.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    $i = 1;\n    while ($i < $n) {\n        $star_num += 6 * $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp4g7bhz1b.php:18\nStack trace:\n#0 /tmp/tmp4g7bhz1b.php(23): test()\n#1 {main}\n  thrown in /tmp/tmp4g7bhz1b.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(3);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "123\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpv95m8lfk.php:17\nStack trace:\n#0 /tmp/tmpv95m8lfk.php(22): test()\n#1 {main}\n  thrown in /tmp/tmpv95m8lfk.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 1;\n    while ($i < $n) {\n        $star_num += 6 * $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(3);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "19\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpo1mwnvam.php:19\nStack trace:\n#0 /tmp/tmpo1mwnvam.php(24): test()\n#1 {main}\n  thrown in /tmp/tmpo1mwnvam.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star = 1;\n    while ($i < $n) {\n        $star += 4;\n        $i++;\n    }\n    return $star;\n}\n\necho find_star_num(1);\necho find_star_num(2);\necho find_star_num(3);\necho find_star_num(4);\necho find_star_num(5);\necho find_star_num(6);\necho find_star_num(7);\necho find_star_num(8);\necho find_star_num(9);\necho find_star_num(10);\necho find_star_num(11);\necho find_star_num(12);\necho find_star_num(13);\necho find_star_num(14);\necho find_star_num(15);\necho find_star_num(16);\necho find_star_num(17);\necho find_star_num(18);\necho find_star_num(19);\necho find_star_num(20);\necho find_star_num(21);\necho find_star_num(22);\necho find_star_num(23);\necho find_star_num(24);\necho find_star_num(25);\necho find_star_num(26);\necho find_star_num(27);\necho find_star_num(28);\necho find_star_num(29);\necho find_star_num(30);\necho find_star_num(31);\necho find_star_num(32);\necho find_star_num(33);\necho find_star_num(34);\necho find_star_num(35);\necho find_star_num(36);\necho find_star_num(37);\necho find_star_num(38);\necho find_star_num(39);\necho find_star_num(40);\necho find_star_num(41);\necho find_star_num(42);\necho find_star_num(43);\necho find_star_num(44);\necho find_star_num(45);\necho find_star_num(46);\necho find_\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nParse error: syntax error, unexpected token \"function\", expecting \",\" or \";\" in /tmp/tmpjo9h_bii.php on line 60\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpd0hemkxh.php:18\nStack trace:\n#0 /tmp/tmpd0hemkxh.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpd0hemkxh.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = ($star_num * 10) + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp7eb7gb7n.php:16\nStack trace:\n#0 /tmp/tmp7eb7gb7n.php(21): test()\n#1 {main}\n  thrown in /tmp/tmp7eb7gb7n.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp313yyb1x.php:19\nStack trace:\n#0 /tmp/tmp313yyb1x.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp313yyb1x.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $count = 0;\n    while ($count < $n) {\n        if (is_star_num($i)) {\n            $count++;\n        }\n        $i++;\n    }\n    return $i - 1;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmp_o2f5e5n.php:7\nStack trace:\n#0 /tmp/tmp_o2f5e5n.php(16): find_star_num(3)\n#1 /tmp/tmp_o2f5e5n.php(20): candidate(3)\n#2 /tmp/tmp_o2f5e5n.php(25): test()\n#3 {main}\n  thrown in /tmp/tmp_o2f5e5n.php on line 7\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp313yyb1x.php:19\nStack trace:\n#0 /tmp/tmp313yyb1x.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp313yyb1x.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $count = 0;\n    while ($count < $n) {\n        if (is_star_num($i)) {\n            $count++;\n        }\n        $i++;\n    }\n    return $i - 1;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmp_o2f5e5n.php:7\nStack trace:\n#0 /tmp/tmp_o2f5e5n.php(16): find_star_num(3)\n#1 /tmp/tmp_o2f5e5n.php(20): candidate(3)\n#2 /tmp/tmp_o2f5e5n.php(25): test()\n#3 {main}\n  thrown in /tmp/tmp_o2f5e5n.php on line 7\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 1;\n    $count = 0;\n    while ($count < $n) {\n        $num++;\n        if (is_star_num($num)) {\n            $count++;\n        }\n    }\n    return $num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Error: Call to undefined function is_star_num() in /tmp/tmp857_8um7.php:8\nStack trace:\n#0 /tmp/tmp857_8um7.php(16): find_star_num(3)\n#1 /tmp/tmp857_8um7.php(20): candidate(3)\n#2 /tmp/tmp857_8um7.php(25): test()\n#3 {main}\n  thrown in /tmp/tmp857_8um7.php on line 8\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\n$n = 5;\necho find_star_num($n);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "12345\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpu9mpws1f.php:18\nStack trace:\n#0 /tmp/tmpu9mpws1f.php(23): test()\n#1 {main}\n  thrown in /tmp/tmpu9mpws1f.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num += $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpp4h_ayin.php:16\nStack trace:\n#0 /tmp/tmpp4h_ayin.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpp4h_ayin.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(3);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "123\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpv95m8lfk.php:17\nStack trace:\n#0 /tmp/tmpv95m8lfk.php(22): test()\n#1 {main}\n  thrown in /tmp/tmpv95m8lfk.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n  $star_num = 0;\n  for ($i = 1; $i <= $n; $i++) {\n    $star_num += $i;\n  }\n  return $star_num;\n}\n\necho find_star_num(5);\necho \"\\n\";\necho find_star_num(10);\necho \"\\n\";\necho find_star_num(20);\necho \"\\n\";\necho find_star_num(50);\necho \"\\n\";\necho find_star_num(100);\necho \"\\n\";\necho find_star_num(200);\necho \"\\n\";\necho find_star_num(500);\necho \"\\n\";\necho find_star_num(1000);\necho \"\\n\";\necho find_star_num(2000);\necho \"\\n\";\necho find_star_num(5000);\necho \"\\n\";\necho find_star_num(10000);\necho \"\\n\";\necho find_star_num(20000);\necho \"\\n\";\necho find_star_num(50000);\necho \"\\n\";\necho find_star_num(100000);\necho \"\\n\";\necho find_star_num(200000);\necho \"\\n\";\necho find_star_num(500000);\necho \"\\n\";\necho find_star_num(1000000);\necho \"\\n\";\necho find_star_num(2000000);\necho \"\\n\";\necho find_star_num(5000000);\necho \"\\n\";\necho find_star_num(10000000);\necho \"\\n\";\necho find_star_num(20000000);\necho \"\\n\";\necho find_star_num(50000000);\necho \"\\n\";\necho find_star_num(100000000);\necho \"\\n\";\necho find_star_num(200000000);\necho \"\\n\";\necho find_star_num(500000000);\necho \"\\n\";\necho find_star_num(1000000000);\necho \"\\n\";\necho find_star_num(2000000000);\necho \"\\n\";\necho find_star_num(5000000000);\necho \"\\n\";\necho find_star_num(10000000000);\necho \"\\n\";\necho find_star_\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nParse error: syntax error, unexpected token \"function\", expecting \",\" or \";\" in /tmp/tmpc6zl0g5z.php on line 70\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpfwa20ins.php:16\nStack trace:\n#0 /tmp/tmpfwa20ins.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpfwa20ins.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpfwa20ins.php:16\nStack trace:\n#0 /tmp/tmpfwa20ins.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpfwa20ins.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num += $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpfrlvvf4p.php:17\nStack trace:\n#0 /tmp/tmpfrlvvf4p.php(22): test()\n#1 {main}\n  thrown in /tmp/tmpfrlvvf4p.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n\t$i = 1;\n\t$star_num = 0;\n\twhile ($i <= $n) {\n\t\t$star_num += $i;\n\t\t$i++;\n\t}\n\treturn $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmprajziffg.php:18\nStack trace:\n#0 /tmp/tmprajziffg.php(23): test()\n#1 {main}\n  thrown in /tmp/tmprajziffg.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    $i = 1;\n    while ($i < $n) {\n        $star_num += 6 * $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(3);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "19\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpwnobf1h3.php:19\nStack trace:\n#0 /tmp/tmpwnobf1h3.php(24): test()\n#1 {main}\n  thrown in /tmp/tmpwnobf1h3.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num += $i;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpfrlvvf4p.php:17\nStack trace:\n#0 /tmp/tmpfrlvvf4p.php(22): test()\n#1 {main}\n  thrown in /tmp/tmpfrlvvf4p.php on line 17\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = ($star_num * 10) + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510667,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpsqy1fzwm.php:16\nStack trace:\n#0 /tmp/tmpsqy1fzwm.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpsqy1fzwm.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n\t$star_num = 0;\n\tfor ($i = 1; $i <= $n; $i++) {\n\t\t$star_num += $i;\n\t}\n\treturn $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpll23es3e.php:16\nStack trace:\n#0 /tmp/tmpll23es3e.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpll23es3e.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpkmef9hd0.php:16\nStack trace:\n#0 /tmp/tmpkmef9hd0.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpkmef9hd0.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = ($star_num * 10) + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp7eb7gb7n.php:16\nStack trace:\n#0 /tmp/tmp7eb7gb7n.php(21): test()\n#1 {main}\n  thrown in /tmp/tmp7eb7gb7n.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $num = (2 * $num) + 1;\n    }\n    return $num;\n}\n\necho find_star_num(1) . \"\\n\";\necho find_star_num(2) . \"\\n\";\necho find_star_num(3) . \"\\n\";\necho find_star_num(4) . \"\\n\";\necho find_star_num(5) . \"\\n\";\necho find_star_num(6) . \"\\n\";\necho find_star_num(7) . \"\\n\";\necho find_star_num(8) . \"\\n\";\necho find_star_num(9) . \"\\n\";\necho find_star_num(10) . \"\\n\";\necho find_star_num(11) . \"\\n\";\necho find_star_num(12) . \"\\n\";\necho find_star_num(13) . \"\\n\";\necho find_star_num(14) . \"\\n\";\necho find_star_num(15) . \"\\n\";\necho find_star_num(16) . \"\\n\";\necho find_star_num(17) . \"\\n\";\necho find_star_num(18) . \"\\n\";\necho find_star_num(19) . \"\\n\";\necho find_star_num(20) . \"\\n\";\necho find_star_num(21) . \"\\n\";\necho find_star_num(22) . \"\\n\";\necho find_star_num(23) . \"\\n\";\necho find_star_num(24) . \"\\n\";\necho find_star_num(25) . \"\\n\";\necho find_star_num(26) . \"\\n\";\necho find_star_num(27) . \"\\n\";\necho find_star_num(28) . \"\\n\";\necho find_star_num(29) . \"\\n\";\necho find_star_num(30) . \"\\n\";\necho find_star_num(31) . \"\\n\";\necho find_star_num(32) . \"\\n\";\necho find_star_num(33) . \"\\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "\nParse error: syntax error, unexpected identifier \"Test\", expecting \",\" or \";\" in /tmp/tmptsek8iqq.php on line 49\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    $i = 1;\n    while ($i < $n) {\n        $star_num = $star_num + 6 * $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(3);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "19\nFatal error: Uncaught Exception: Test failed! in /tmp/tmptfzya36z.php:19\nStack trace:\n#0 /tmp/tmptfzya36z.php(24): test()\n#1 {main}\n  thrown in /tmp/tmptfzya36z.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp313yyb1x.php:19\nStack trace:\n#0 /tmp/tmp313yyb1x.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp313yyb1x.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $num = 0;\n    while ($n > 0) {\n        $num += $n;\n        $n--;\n    }\n    return $num;\n}\n\necho find_star_num(5);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "15\nFatal error: Uncaught Exception: Test failed! in /tmp/tmps6091xko.php:18\nStack trace:\n#0 /tmp/tmps6091xko.php(23): test()\n#1 {main}\n  thrown in /tmp/tmps6091xko.php on line 18\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $i = 1;\n    $star_num = 0;\n    while ($i <= $n) {\n        $star_num += $i;\n        $i++;\n    }\n    return $star_num;\n}\n\necho find_star_num(4);\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123895,
      "stdout": "10\nFatal error: Uncaught Exception: Test failed! in /tmp/tmp3m8gjxn2.php:19\nStack trace:\n#0 /tmp/tmp3m8gjxn2.php(24): test()\n#1 {main}\n  thrown in /tmp/tmp3m8gjxn2.php on line 19\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 1;\n    for ($i = 1; $i < $n; $i++) {\n        $star_num = $star_num * 2 + 1;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1666123894,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpnmyqjt79.php:16\nStack trace:\n#0 /tmp/tmpnmyqjt79.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpnmyqjt79.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    },
    {
      "program": "<?php\n// Write a function to find the n'th star number.\nfunction find_star_num($n) {\n    $star_num = 0;\n    for ($i = 1; $i <= $n; $i++) {\n        $star_num = $star_num * 10 + $i;\n    }\n    return $star_num;\n}\n\nfunction candidate(...$args) {\n    return find_star_num(...$args);\n}\n\nfunction test(): void {\n    if (candidate(3) !== 37) { throw new Exception(\"Test failed!\"); }\n    if (candidate(4) !== 73) { throw new Exception(\"Test failed!\"); }\n    if (candidate(5) !== 121) { throw new Exception(\"Test failed!\"); }\n}\n\ntest();",
      "timestamp": 1665510666,
      "stdout": "\nFatal error: Uncaught Exception: Test failed! in /tmp/tmpvg4_x_tu.php:16\nStack trace:\n#0 /tmp/tmpvg4_x_tu.php(21): test()\n#1 {main}\n  thrown in /tmp/tmpvg4_x_tu.php on line 16\n",
      "stderr": "",
      "exit_code": 255,
      "status": "Exception"
    }
  ]
}